webpack.config.js

```diff
@@ -24,20 +24,31 @@ module.exports = {
   entry: './src/io/react-native/react-native-worker.js',
   mode: production ? 'production' : 'development',
   module: {
-    rules: [
-      {
-        test: /\.js$/,
-        exclude: /node_modules/,
-        use: { loader: 'babel-loader', options: babelOptions }
-      },
-      {
-        include: path.resolve(__dirname, 'node_modules/buffer/index.js'),
-        use: {
-          loader: 'babel-loader',
-          options: { presets: ['@babel/preset-env'] }
-        }
-      }
-    ]
+    rules: production
+      ? [
+          {
```

Comment:
Please only replace the first thing in the array. The second one needs to be enabled in both modes.